docs/index.md

```diff
@@ -47,7 +47,7 @@ This project was initially inspired by a [gist](https://gist.github.com/mosheesh
 	* discuss with the authors on an issue ticket prior to doing anything big
 	* follow the style, naming and structure conventions of the rest of the project
 	* make commits atomic and easy to merge
-	* verify all tests are passing. Build the project with `mvn clean install -Pproprietary-deps` to do this.
+	* verify all tests are passing. Build the project with `./gradlew test` to do this.
```

Comment:
Please could we mention the build cache here?

People might want to re-run all tests just to be sure, especially, for example, if they're trying to check both Docker Machine and Docker for * compatibility. It would probably be helpful to explain how to disable the cache. (Is `./gradlew test --no-build-cache` correct?)

Reply:
good idea, I'll add it 👍

Reply:
AFAIR the recommended test task should be `check`. `check` dependsOn `test` by default. I would document `check` here, so that future test tasks (e.g. TestNG oder integrative tests), can be introduced without telling people about another test task.

Reply:
Yes, you are right. I think we should change the usage in Travis and CircleCI accordingly also.